pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -172,7 +172,7 @@ async def run(self):
                     await self.slack_client.say_in_thread(message)
                 except Exception as err:
                     self._logger.warning("Failed to trigger microshift_sync job: %s", err)
-                    message = "@release-artists Please start <https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888" \
+                    message = "@release-artists Please start <https://art-jenkins.apps.prod-stable-spoke1-dc-iad2.itup.redhat.com" \
```

Comment:
suggest using constants.JENKINS_SERVER_URL instead of hardcode

Reply:
done